modules/gapi/test/serialization/gsrlzcore.cpp

```diff
@@ -0,0 +1,652 @@
+    {
+        in.convertTo(out, rtype, alpha, beta);
+    }
+};
+
+GAPI_SRLZ_KERNEL(GSRLZSqrt, cv::gapi::core::GSqrt)
+{
+    static void run(const cv::Mat& in, cv::Mat &out)
+    {
+        cv::sqrt(in, out);
+    }
+};
+
+GAPI_SRLZ_KERNEL(GSRLZNormalize, cv::gapi::core::GNormalize)
+{
+    static void run(const cv::Mat& src, double a, double b,
+                    int norm_type, int ddepth, cv::Mat& out)
+    {
+        cv::normalize(src, out, a, b, norm_type, ddepth);
+    }
+};
+
+GAPI_SRLZ_KERNEL(GSRLZWarpPerspective, cv::gapi::core::GWarpPerspective)
+{
+    static void run(const cv::Mat& src, const cv::Mat& M,  const cv::Size& dsize,
+                    int flags, int borderMode, const cv::Scalar& borderValue, cv::Mat& out)
+    {
+        cv::warpPerspective(src, out, M, dsize, flags, borderMode, borderValue);
+    }
+};
+
+GAPI_SRLZ_KERNEL(GSRLZWarpAffine, cv::gapi::core::GWarpAffine)
+{
+    static void run(const cv::Mat& src, const cv::Mat& M,  const cv::Size& dsize,
+                    int flags, int borderMode, const cv::Scalar& borderValue, cv::Mat& out)
+    {
+        cv::warpAffine(src, out, M, dsize, flags, borderMode, borderValue);
+    }
+};
+
+
+cv::gapi::GKernelPackage cv::gapi::core::serialization::kernels()
+{
+    static auto pkg = cv::gapi::kernels
+        <  GSRLZAdd
+         , GSRLZAddC
+         , GSRLZSub
+         , GSRLZSubC
+         , GSRLZSubRC
+         , GSRLZMul
+         , GSRLZMulC
+         , GSRLZMulCOld
+         , GSRLZDiv
+         , GSRLZDivC
+         , GSRLZDivRC
+         , GSRLZMean
+         , GSRLZMask
+         , GSRLZPolarToCart
+         , GSRLZCartToPolar
+         , GSRLZPhase
+         , GSRLZCmpGT
+         , GSRLZCmpGE
+         , GSRLZCmpLE
+         , GSRLZCmpLT
+         , GSRLZCmpEQ
+         , GSRLZCmpNE
+         , GSRLZCmpGTScalar
+         , GSRLZCmpGEScalar
+         , GSRLZCmpLEScalar
+         , GSRLZCmpLTScalar
+         , GSRLZCmpEQScalar
+         , GSRLZCmpNEScalar
+         , GSRLZAnd
+         , GSRLZAndS
+         , GSRLZOr
+         , GSRLZOrS
+         , GSRLZXor
+         , GSRLZXorS
+         , GSRLZNot
+         , GSRLZSelect
+         , GSRLZMin
+         , GSRLZMax
+         , GSRLZAbsDiff
+         , GSRLZAbsDiffC
+         , GSRLZSum
+         , GSRLZAddW
+         , GSRLZNormL1
+         , GSRLZNormL2
+         , GSRLZNormInf
+         , GSRLZIntegral
+         , GSRLZThreshold
+         , GSRLZThresholdOT
+         , GSRLZInRange
+         , GSRLZSplit3
+         , GSRLZSplit4
+         , GSRLZResize
+         , GSRLZResizeP
+         , GSRLZMerge3
+         , GSRLZMerge4
+         , GSRLZRemap
+         , GSRLZFlip
+         , GSRLZCrop
+         , GSRLZCopy
+         , GSRLZConcatHor
+         , GSRLZConcatVert
+         , GSRLZLUT
+         , GSRLZConvertTo
+         , GSRLZSqrt
+         , GSRLZNormalize
+         , GSRLZWarpPerspective
+         , GSRLZWarpAffine
+         >();
+    return pkg;
+}
```

Comment:
Please remove all the kernels which are not required for your pipelines

Reply:
Done